### PR TITLE
Update activities version

### PR DESCRIPTION
**Pull request by svanherk**
Also a table fix for the "light" style

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -2662,7 +2662,7 @@
       }
     },
     "d2l-hypermedia-constants": {
-      "version": "github:Brightspace/d2l-hypermedia-constants#05a64ab9c333db19b5ec64b757a67c5fccaf27f8",
+      "version": "github:Brightspace/d2l-hypermedia-constants#389ee800e6b1907a259fe75bb014db42a7810037",
```
Comment (svanherk) [inline package-lock.json]: FYI @ryantmer

File: package-lock.json [hunk 2]
```diff
@@ -2974,7 +2974,7 @@
       }
     },
     "d2l-resize-aware": {
-      "version": "git://github.com/BrightspaceUI/resize-aware.git#eea8673eafd1c0deb1ac8d1ee05037c02cf71aaa",
+      "version": "git://github.com/BrightspaceUI/resize-aware.git#3a6e0b3bb6f8befc0e2a80c7295dfa3e33dbcdd2",
```
Comment (svanherk) [inline package-lock.json]: FYI @mpharoah-d2l

File: package-lock.json [hunk 3]
```diff
@@ -1905,9 +1905,9 @@
       }
     },
     "caniuse-lite": {
-      "version": "1.0.30000945",
-      "resolved": "https://registry.npmjs.org/caniuse-lite/-/caniuse-lite-1.0.30000945.tgz",
-      "integrity": "sha512-PSGwYChNIXJ4FZr9Z9mrVzBCB1TF3yyiRmIDRIdKDHZ6u+1jYH6xeR28XaquxnMwcZVX3f48S9zi7eswO/G1nQ==",
```
Comment (mtjandra) [inline package-lock.json]: Unsure why this updated, but I think should be fine?

File: package-lock.json (hunk 1 above)
Comment (ryantmer) [reply package-lock.json]: Yep all good - just added a new rel.

File: package-lock.json (hunk 2 above)
Comment (mpharoah-d2l) [reply package-lock.json]: I was going to update this anyways. Saves me a PR

**Pull request merged**